src/EmptyMessage /EmptyMessage.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const EmptyMessage = ({type}) => (
+  <div>
+    <p>No {type} yet. <Link to={`${type}/create`}>Create new {type.substring(0, type.length - 1)}</Link></p>
```

Comment:
Looking good 👍 This wouldn't work for the different types of copies/'entities' though, that _don't_ have 's' at the end e.g
- Journalism
- Gigs
- Collaborators 

'Journalism' would become 'Journalis' 🙃 

Could add some logic to get around this, but I think it'd be simpler to just pass a prop in, `createCopy` then it's customisable from anywhere:

```
const uppercaseEntityName = entityName[0].toUpperCase() + entityName.slice(1);
... 
<p>No {entityName} yet. <Link to={`${entityName}/create`}>{createCopy}</Link></p>
```

Also if we capitilise the first letter for the first part of the string, then it's consistent with other design/copy, and we only have to pass one prop in, so usage would be:

```
<EmptyMessage
  entityName='journalism'
  createCopy='Create a new Journalism article'
/>
```

and others would be like:
```
createCopy='Create a new Gig'
createCopy='Create a new Collaborator'
createCopy='Create News'
```


What do you think?

Reply:
Yes, I did not think of Journalism. If I was adding 's' to it, it would not work either. Well spotted!

I do not mind, I can do with the createCopy or create an exception.